src/web/app/components/question-types/question-statistics/num-scale-question-statistics.component.ts

```diff
@@ -48,6 +48,9 @@ export class NumScaleQuestionStatisticsComponent extends NumScaleQuestionStatist
     this.rowsData = [];
     for (const team of Object.keys(this.teamToRecipientToScores)) {
       for (const recipient of Object.keys(this.teamToRecipientToScores[team])) {
+        if (this.isStudent && recipient !== 'You') {
```

Comment:
I don't think the filtering should happen here as the statistics component's responsibility is just display stats based on provided responses. It essentially should not do the filtering.